packages/mobile/src/account/Education.tsx

```diff
@@ -28,19 +38,35 @@ interface EducationStep {
   image: ImageSourcePropType | null
   topic: EducationTopic
   title: string
-  text: string
+  // If set to true, title is displayed at the top
+  isTopTitle?: boolean
```

Comment:
`useTopTitle`? or `showTopTitle`?

Reply:
`is*` is pretty common way to name booleans. I dont follow the suggestion?

Reply:
Yes but given that's its a prop, it implies *this* component is a top title. But it's just a flag for where to place to title